ee/backend/controllers/bitbucket.go

```
@@ -0,0 +1,340 @@
+	if len(jobs) == 0 {
+		log.Printf("no projects impacated, succeeding")
+		// This one is for aggregate reporting
+		err = utils.SetPRStatusForJobs(bbService, issueNumber, jobs)
+		return nil
+	}
+
+	err = utils.SetPRStatusForJobs(bbService, issueNumber, jobs)
+	if err != nil {
+		log.Printf("error setting status for PR: %v", err)
+		utils.InitCommentReporter(bbService, issueNumber, fmt.Sprintf(":x: error setting status for PR: %v", err))
+		fmt.Errorf("error setting status for PR: %v", err)
```

golangci-lint (1.62.2)

Error return value of `fmt.Errorf` is not checked

(errcheck)